src/index.js

```diff
@@ -1,3 +1,8 @@
-const run = require('./run').run
+const core = require('@actions/core')
+const run = require('./run')
 
-run()
+try {
+  run()
```

Comment:
@jveldboom Does this need an `await`?

```suggestion
  await run()
```

Reply:
Good catch - yes it does. But I'm not 100% sure if the context, where the action runs, is already within an `async` context so a top-level `await` is available.

I'll need to do a little testing to validate this. I'll update the PR with the findings.

Reply:
Well I'm not still not 100% sure if the GitHub Action runs within an `async` context but the dist build process (`yarn build`) fails due to it knowing the top-level `await` is not available. Here's the error when building with the suggested code change.
```
ncc: Version 0.36.0
ncc: Compiling file index.js into CJS
Error: Module parse failed: Top-level-await is only supported in EcmaScript Modules
```

I've just updated the PR to include an anonymous `async` function which will allow us to write a simple `try/catch` with an `await`. Not sure I'm still happy with it but it does work for now. 🤷 

Here's a testing workflow where I forced an exception to validate it was caught properly and only the `err.message` was output. https://github.com/jveldboom/action-conventional-versioning/actions/runs/4158637917/jobs/7193992823#step:4:7

Reply:
Thanks for entertaining me. In hindsight, you've helped me question my suggestion to add an `await`. However, wrapping the `await run()` in a self-executing anonymous `async` function sounds good, too!